src/main/resources/static/openapi/SupportAdditionalNeedsAPI.yml

```diff
@@ -1035,12 +1040,6 @@ components:
         prisonId:
           $ref: '#/components/schemas/PrisonId'
           description: The prisonId that the person is a resident at at the time of creating the review.
-        reviewDate:
```

Comment:
Minor, but I think this property is still referenced in the `required` array below?